apps/dashboard/src/data/editor/reducer.js

```diff
@@ -42,7 +47,28 @@ const hasUnsavedChanges = ( state = false, action ) => {
 	return state;
 };
 
+const title = ( state = '', action ) => {
+	if ( action.type === EDITOR_TITLE_SET ) {
+		return action.title;
+	}
+
+	return state;
+};
+
+const content = ( state = '', action ) => {
+	if (
+		action.type === EDITOR_CONTENT_UPDATE ||
+		action.type === EDITOR_CONTENT_RESTORE
```

Comment:
I think `RESTORE` should just set this to `''` rather than `content`, right ? Or not do anything at all.

Reply:
The restore receives the new content from the project.draftContent (meaning, it restores to whatever you provide), the main difference is that a restore (like an initial load) skips triggering the isolated-editor `onSaveContent` callback.